glow_test.go

```diff
@@ -61,3 +65,37 @@ func TestGlowFlags(t *testing.T) {
 		}
 	}
 }
+
+func TestPagerCommandValues(t *testing.T) {
+	tt := []struct {
+		pagerEnvVar     string
+		expectedCommand []string
+	}{
+		{
+			pagerEnvVar:     "C:\\Program Files\\Git\\usr\\bin\\less.exe",
+			expectedCommand: []string{"C:\\Program Files\\Git\\usr\\bin\\less.exe"},
+		},
+		{
+			pagerEnvVar:     "usr/local/bin",
+			expectedCommand: []string{"usr/local/bin"},
+		},
+		{
+			pagerEnvVar:     "",
+			expectedCommand: []string{"less", "-r"},
+		},
+		{
+			pagerEnvVar:     "",
+			expectedCommand: []string{"less", "-r"},
+		},
+	}
+
+	for _, v := range tt {
+		pager := "PAGER"
+		os.Setenv(pager, v.pagerEnvVar)
+		os.Setenv("PATH", os.Getenv("PATH")+":"+v.pagerEnvVar)
```

Comment:
Do we actually need to update the PATH here? It will get expanded further with each iteration.

Reply:
Not required, had set it for testing a previous iteration, removed.